### PR TITLE
Improve Gif block previews

### PR DESCRIPTION
**Pull request by jlemay-godaddy**
### Description
Improve Gif block previews, mainly using 100% container width and keeping aspect ratio

### Screenshots
image

### Types of changes
Visual changes

### How has this been tested?
Manually tested

### Checklist:
- [x] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

**Review by AnthonyLedesma — APPROVED** [review]
CSS Wizard 🧙  LGTM

**Comment by jlemay-godaddy** [comment]
@AnthonyLedesma I'll let you merge, I don't have write access with this account!

**Pull request merged**
